VERSIONING.md

```diff
@@ -12,76 +12,92 @@ Minor releases contain new smaller features. Minor releases are fully backward-c
-- {Type}: Letter indicating the type of issue addressed on the branch
-    - F: New feature
-    - B: Bug
-- {Issue#}: GitHub issue number associated to the feature or defect to be addressed on the branch. That would allow better tracking between branches and change description.
-- {ShortTitle}: Short title of the change. Written using [UpperCamelCase](https://es.wikipedia.org/wiki/CamelCase)
+## Git FLow
 
-Examples
-- master-B12-DisableDragAndDrop
-- 4.1.x-F68-RoleDirective
+In the Git flow development model, one main development branch is considered.
+
+Feature branches are created from de development branch and once they are done, pull requests are created.
+
+Once the pull requests are accepted and merged to the main branch, and it’s decided that the main branch has reached enough maturity to be released,
+a separate branch is created to prepare a final version.
+In this branch the application is tested and fixing is done. After this, the branch is merged with the master branch and tag it with the release version.
+In the meantime, new features can be developed on the develop branch.
+
+## Trunk-based Development Workflow
+
+In the trunk-based development model, all developers work on a single branch with open access to it.
+
+Short-lived feature branches are created in order to develop new features, and once code on their branch compiles and passes all tests, the developers merge it straight to master. It ensures that development is truly continuous and prevents developers from creating merge conflicts that are difficult to resolve.
 
+# Angular libraries
 
-## Maintenance branches
+## Branching
+Trunk-based development model has been selected for branching.
 
-After a feature release, it is needed to manage a maintenance branch.
+### Feature branches
+
+Feature branches should be named according to the following pattern:
+
+feature-{IssueShortTitle}
+
+where {ShortTitle} is written using [UpperCamelCase](https://es.wikipedia.org/wiki/CamelCase)
+
+Examples
+- feature-DisableDragAndDrop
+- feature-RoleDirective
 
-First, if you wish to continue to release maintenance fixes for the feature release made before the recent one, then you must create another branch to track commits for that previous release.
+### Release branches
 
-To do this, the feature branch is copied to another branch named with the previous release version number.
+After a release, it is needed to manage a release branch.
 
-Maintenance branches will be named as Major.Minor.x, for example 5.12.x
+Release branches will be named as release-Major.Minor.x, for example release-5.12.x
 
-Rules to 'Require pull request reviews before merging' must be applied to every maintenance branch. You can use wildcards in order to do so: \[1-9]\*.* will do the work
+Rules to 'Require pull request reviews before merging' must be applied to every release branch. You can use wildcards in order to do so: release-* will do the work
```

Comment:
I would suggest defining here the concept of "active release branch" as the release branch used by at least one project. of the company. I think that it will be helpful for stating the bug fixing policy.

Reply:
OK